apps/sensenet/src/components/dialogs/add.tsx

```diff
@@ -26,7 +26,7 @@ export const AddDialog: React.FunctionComponent<AddDialogProps> = ({ schema, par
           handleCancel={closeLastDialog}
           repository={repo}
           contentTypeName={schema.ContentTypeName}
-          path={parent.Path}
+          path={parent}
```

Comment:
While this is not part of the pr, passing down the path to NewView only to have it back on onSubmit feels wrong IMO.

Reply:
this was necessary because of dms and the redux actions, but now I removed it from here and add the path at the related submit in the dms